routes/api/homepage.js

```diff
@@ -0,0 +1,49 @@
+const express = require('express');
+const router = express.Router();
+const mongoose = require('mongoose');
+const passport = require('passport');
+const Post = require('../../models/Posts');
+const User = require('../../models/User');
+
+router.get('/home', passport.authenticate('jwt', { session: false }), (req, res) => {
+  var userFollowing = [];
+  var myPost = mongoose.Types.ObjectId(req.user.id);//get logged in user id to get post
```

Comment:
What is the difference between myPost and req.user.id? They both hold the same value.

Also, 'myPost' variable name is confusing, myId or myUserId might be good.

Reply:
I removed variable

Reply:
removed the req.user.id from the array.